scripts/update_configs.py

```diff
@@ -86,23 +100,94 @@ def _apply_changes_based_on_config(
         writable_local_file.write('\n'.join(local_lines) + '\n')
 
 
-def _update_configs():
-    """Updates the 'feconf.py' and 'constants.ts' files after doing the
+def check_updates_to_terms_of_service():
+    """Checks if updates are made to terms of service and updates
+    REGISTRATION_PAGE_LAST_UPDATED_UTC in feconf.py if there are updates.
+    """
+    personal_access_token = common.get_personal_access_token()
+    g = github.Github(personal_access_token)
+    repo = g.get_organization('oppia').get_repo('oppia')
+
+    common.open_new_tab_in_browser_if_possible(
+        'https://github.com/oppia/oppia/commits/develop/core/'
+        'templates/dev/head/pages/terms-page/terms-page.mainpage.html')
+    python_utils.PRINT(
+        'Are the terms of service changed? Check commits/changes made '
+        'to the file: terms-page.mainpage.html. Enter y/ye/yes if they '
+        'are changed else enter n/no.')
+    is_terms_of_service_changed = python_utils.INPUT().lower()
+    while is_terms_of_service_changed not in ['y', 'ye', 'yes', 'n', 'no']:
+        python_utils.PRINT(
+            'Invalid Input: %s. Please enter yes or no.' % (
+                is_terms_of_service_changed))
+        is_terms_of_service_changed = python_utils.INPUT().lower()
+
+    if is_terms_of_service_changed in feconf.AFFIRMATIVE_CONFIRMATIONS:
+        python_utils.PRINT(
+            'Enter sha of the commit which changed the terms of service.')
+        commit_sha = python_utils.INPUT().lstrip().rstrip()
+        commit_time = repo.get_commit(commit_sha).commit.committer.date
+        time_tuple = (
+            commit_time.year, commit_time.month, commit_time.day,
+            commit_time.hour, commit_time.minute, commit_time.second)
+        feconf_lines = []
+        with python_utils.open_file(LOCAL_FECONF_PATH, 'r') as f:
+            feconf_lines = f.readlines()
+        with python_utils.open_file(LOCAL_FECONF_PATH, 'w') as f:
+            for line in feconf_lines:
+                if line.startswith('REGISTRATION_PAGE_LAST_UPDATED_UTC'):
+                    line = (
+                        'REGISTRATION_PAGE_LAST_UPDATED_UTC = '
+                        'datetime.datetime(%s, %s, %s, %s, %s, %s)\n' % (
+                            time_tuple))
+                f.write(line)
+
+
+def add_mailgun_api_key():
+    """Adds mailgun api key to feconf.py."""
+    mailgun_api_key = getpass.getpass(
+        prompt=('Enter mailgun api key from the release process doc.'))
+
+    if re.match('^key-[a-z0-9]{32}$', mailgun_api_key) is None:
+        raise Exception('Invalid mailgun api key.')
+
+    feconf_lines = []
+    with python_utils.open_file(LOCAL_FECONF_PATH, 'r') as f:
+        feconf_lines = f.readlines()
+
+    feconf_lines.append('MAILGUN_API_KEY = \'%s\'\n' % mailgun_api_key)
```

Comment:
We should replace the existing line, I think, not append a new one. Shouldn't declare the same var twice in a file. And if the expected line isn't present we should throw an error.

Reply:
No such line is present initially in feconf. Should I add a test instead which checks this line is not present?

Reply:
Such a line does exist. It is located [here](https://github.com/oppia/oppia/blob/develop/feconf.py#L355).

Reply:
Oh, Sorry. I missed that. Updated the code now.